src/organization-domains/fixtures/get-organization-domain-pending.json

```diff
@@ -0,0 +1,8 @@
+{
+    "object": "organization_domain",
+    "id": "org_domain_01HD50K7EPWCMNPGMKXKKE14XT",
+    "domain": "workos.com",
+    "state": "pending",
+    "verification_token": "F06PGMsZIO0shrveGWuGxgCj7"
+  }
+  
```

Comment:
Is this end line in the JSON intentional, do we have some linting rule for that?

Reply:
👍  remove the end line and reformatted the json payload.